frontend/src/components/Payment/QRCodeScanner.tsx

```
@@ -0,0 +1,267 @@
+  const [error, setError] = useState<string | null>(null);
+  const [detectedPayment, setDetectedPayment] = useState<ParsedStellarPaymentURI | null>(null);
+  const [rawValue, setRawValue] = useState<string | null>(null);
+
+  const stopCamera = useCallback(() => {
+    if (animationFrameRef.current !== null) {
+      cancelAnimationFrame(animationFrameRef.current);
+      animationFrameRef.current = null;
+    }
+
+    if (streamRef.current) {
+      streamRef.current.getTracks().forEach((track) => track.stop());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)